config/pipelines/high_throughput_heron_384_tailed.yml

```diff
@@ -0,0 +1,26 @@
+---
+Heron-384 Tailed A: # Heron 384-well pipeline specific to PCR 1 plate
+  filters:
+    request_type_key: limber_heron_lthr
+    library_type: PCR amplicon tailed adapters 384
+  library_pass: LTHR-384 Lib PCR pool
+  relationships:
+    LTHR-384 RT: LTHR-384 PCR 1
+    LTHR-384 PCR 1: LTHR-384 Lib PCR 1
+    LTHR-384 Lib PCR 1: LTHR-384 Lib PCR pool
+Heron-384 Tailed B:
+  filters:
+    request_type_key: limber_heron_lthr
+    library_type: PCR amplicon tailed adapters 384
```

Comment:
Is that library type correct for tailed?, looks same as other heron pipeline. Are the libraries essentially the same whether constructed by either pipeline?

Reply:
It has the word tailed, in place of ligated.